### PR TITLE
feat: fix imports in tests

### PR DESCRIPTION
**Pull request by amacar**
### What changes are being made?
Use imports from ARK exported modules. Fixes https://github.com/protokol/nft-plugins/issues/145

### Why are these changes necessary?
To support migration to Yarn v2.

## Checklist

- [ ] Documentation _(if necessary)_
- [x] Tests _(if necessary)_
- [x] Ready to be merged

### CODE REVIEW

File: packages/nft-exchange-transactions/__tests__/functional/transaction-forging/__support__/transaction-factory.ts [hunk 1]
```diff
@@ -40,4 +41,28 @@ export class NFTExchangeTransactionFactory extends TransactionFactory {
 
         return this;
     }
+
+    public NFTRegisterCollection(nftCollection: NFTInterfaces.NFTCollectionAsset): NFTExchangeTransactionFactory {
```
Comment (KovacZan) [inline packages/nft-exchange-transactions/__tests__/functional/transaction-forging/__support__/transaction-factory.ts:45]: @amacar wouldn't be better if you pull transaction-factory from nft-base, avoiding code duplication?

Comment (amacar) [reply packages/nft-exchange-transactions/__tests__/functional/transaction-forging/__support__/transaction-factory.ts:45]: Yes, but then we need to export it from the tests folder.

Comment (KovacZan) [reply packages/nft-exchange-transactions/__tests__/functional/transaction-forging/__support__/transaction-factory.ts:45]: Okej is not that much code, we can leave it as it is.

**Review by KovacZan — APPROVED** [review]

**Review by kristjank — APPROVED** [review]

File: packages/nft-base-transactions/__tests__/unit/__support__/app.ts [hunk 2]
```diff
@@ -1,36 +1,22 @@
...
-    ipfsIndexer,
-    locksIndexer,
-    publicKeysIndexer,
-    usernamesIndexer,
-} from "@arkecosystem/core-state/src/wallets/indexers/indexers";
-import { Mocks } from "@arkecosystem/core-test-framework";
-import { Generators } from "@arkecosystem/core-test-framework/src";
-import { Collator } from "@arkecosystem/core-transaction-pool/src";
+import { Stores, Wallets } from "@arkecosystem/core-state";
+import { addressesIndexer, publicKeysIndexer } from "@arkecosystem/core-state/src/wallets/indexers/indexers";
```
[the remark this answers is not in the document]
Comment (amacar) [reply packages/nft-base-transactions/__tests__/unit/__support__/app.ts:3]: Yes, need to update our core first.

**Pull request merged**